azure-webapp-maven-plugin/README.md

```diff
@@ -221,172 +226,32 @@ Property | Description
 `includes` | A list of patterns to include, e.g. `**/*.war`.
 `excludes` | A list of patterns to exclude, e.g. `**/*.xml`.
 
-> Note: The `<targetPath>` is relative to the `/site/wwwroot/` folder except one case: it is relative to the 
+> Note: The `<targetPath>` is relative to the `/site/wwwroot/` folder except one case: it is relative to the
 `/site/wwwroot/webapps` when you deploy the war package.
 
 #### `<region>`
-All valid regions are listed as below. Read more at [Azure Region Availability](https://azure.microsoft.com/en-us/regions/services/).
-- `westus`
-- `westus2`
-- `centralus`
-- `eastus`
-- `eastus2`
-- `northcentralus`
-- `southcentralus`
-- `westcentralus`
-- `canadacentral`
-- `canadaeast`
-- `brazilsouth`
-- `northeurope`
-- `westeurope`
-- `uksouth`
-- `ukwest`
-- `eastasia`
-- `southeastasia`
-- `japaneast`
-- `japanwest`
-- `australiaeast`
-- `australiasoutheast`
-- `centralindia`
-- `southindia`
-- `westindia`
-- `koreacentral`
-- `koreasouth`
+Some valid regions are listed as below. All valid regions pelease refer to [Azure Region Availability](https://azure.microsoft.com/en-us/global-infrastructure/services/?regions=all&products=app-service).
+- `West US`
```

Comment:
we could provide the region as well as its value, like tables in `<webContainer>`

Supported Value | Description
--- | ---
westus | West US

Reply:
The supported region may change, we have to keep eye on the changes if we list all regions. So here we just provide the link

Reply:
I think Flanker32 means you could list samples like support value and its description. Does our maven plugin accept "West US" and "westus"? If yes, it does not matter. If we only accept "westus", you may want to use such a format.

Reply:
yes, "west us" and "westus" are all accepted.